pmm/pmm2-ovf-test.groovy

```diff
@@ -117,29 +129,43 @@ pipeline {
                     
                     tar xvf \$VM_NAME.ova
                     export ovf_name=$(find -type f -name '*.ovf');
+                    ifconfig
+                    ip addr show
+                    sudo yum install -y NetworkManager
+                    systemctl start NetworkManager
+                    service NetworkManager-wait-online.service start
+                    systemctl status NetworkManager
                     VBoxManage import \$ovf_name --vsys 0 --memory 2048 --vmname \$VM_NAME > /dev/null
                     VBoxManage modifyvm \$VM_NAME \
                         --memory 2048 \
                         --audio none \
-                        --nic1 bridged --bridgeadapter1 bond0 \
+                        --natpf1 "guestssh,tcp,,80,,80" \
                         --uart1 0x3F8 4 --uartmode1 file /tmp/\$VM_NAME-console.log \
                         --groups "/\$OWNER,/${JOB_NAME}"
+                    VBoxManage modifyvm \$VM_NAME --natpf1 "guesthttps,tcp,,443,,443"
+                    for p in $(seq 0 10); do
+                        VBoxManage modifyvm \$VM_NAME --natpf1 "guestexporters\$p,tcp,,4200\$p,,4200\$p"
+                    done
                     VBoxManage startvm --type headless \$VM_NAME
-
+                    sleep 180
```

Comment:
can we remove it?

Reply:
OVF takes some time to get running, I try to give it some space to bread, so I think is useful.